docs/VCI-trust-framework-mapping.md

```diff
@@ -0,0 +1,236 @@
+# Health Cards to Trust Framework Model Mapping
+
+This document is written in language that should be clear to identity providers participating in the Decentralized Identity ecosystem.
+
+## Health Cards
+
+While Health-Cards utilizes [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/) and [Decentralized Identifiers](https://w3c.github.io/did-core/), it is not a decentralized identity system. The overall approach is to wrap FHIR 4.0 data in a Verifiable Credential (VC) format [(JWS)](https://tools.ietf.org/html/rfc7515) which is signed by the issuer of the VC. This data will be ingestible by any Health-Cards compatible Healthcare Provider (HCP). The Verifiable Credential does not authenticate the holder utilizing the VC, but instead, patient matching is done with the FHIR 4.0 API compatible patient record wrapped in the VC. This method allows data transport and control by the patient with minimal impact on existing systems and processes.
```

Comment:
Health-Cards do not use DIDs AFAIK

Reply:
Work on the trust framework mapping document stalled in early 2021; @cjbuchanan should we remove this document from the repo?

Reply:
@cjbuchanan I think this is at your discretion; this isn't something our implementer community has been clamoring for, but I'd be happy to include it if you think it'd be valuable.